### ISSUE TITLE
Disabling scroll is not working on mobile devices (iPhone 5)

### ISSUE THREAD

**Issue by fuongbb** [jasny/bootstrap#299]
Hi,

Disabled scroll is set to "true" as default. But it is still to scroll content. I made some testing on my iPhone 5.

Any solutions to fix this?
Thanks!

**Comment by hohl** [comment]
I'm still struggling with this problem. In fact my problem is, that when the user "over"scrolls the reveal menu the iPhones Safari starts to scroll the main (body) view. Any chances to prevent that?

**Comment by jonathonmills** [comment]
This fixed the issue for me (also using the existing bootstrap, as the modal backdrop was required.)

```javascript
$('#mobileMenu').on('show.bs.offcanvas', function() {
    $('<div class="modal-backdrop"></div>').appendTo(document.body);
    $('body').bind('touchmove', function(e){e.preventDefault()})
});

$('#mobileMenu').on('hide.bs.offcanvas', function() {
    $(".modal-backdrop").remove();
    $('body').unbind('touchmove')
});
```

**Comment by jasny** [comment]
Fixed by #521
